src/document/previousSpecs/v0_3_1.ts

```diff
@@ -0,0 +1,103 @@
+/** CHANGES from v0.3:
+ * added event markers
+ */
+
+export const SAVE_FILE_VERSION = "v0.3.1";
+export interface SavedWaypoint {
+  x: number;
+  y: number;
+  heading: number;
+  isInitialGuess: boolean;
+  translationConstrained: boolean;
+  headingConstrained: boolean;
+  controlIntervalCount: number; // positive-integer>
```

Comment:
make comments more uniform
```suggestion
  controlIntervalCount: number; // positive-integer
```